epoxy-processor/src/main/java/com/airbnb/epoxy/ModelViewWriter.kt

```diff
@@ -214,9 +217,14 @@ internal class ModelViewWriter(
             boundObjectParam: ParameterSpec,
             viewAttribute: ViewAttributeInfo
     ): CodeBlock {
-        return CodeBlock.of("\$L.\$L(\$L);\n", boundObjectParam.name,
-                            viewAttribute.viewSetterMethodName,
-                            getValueToSetOnView(viewAttribute, boundObjectParam))
+        val expression = if (viewAttribute.viewAttributeTypeName == ViewAttributeType.Field)
+            "\$L.\$L=\$L;\n" else "\$L.\$L(\$L);\n"
```

Comment:
I think we can do `CodeBlock.builder().addStatement()...` to avoid having to do `;\n` manually.